### PR TITLE
  Document dev environment in README

### PR DESCRIPTION
**Pull request by hoosteeno**

### CODE REVIEW

**Comment by pdehaan** [comment]
Looks like we'll need to ignore the ./dev_profile directory from the **standard** linter.

@hoosteeno Can you try adding the following to package.json file and see if that fixes Travis-CI:

```
"standard": {
  "ignore": [
    "dev_profile"
  ]
}
```
via http://standardjs.com/#how-do-i-ignore-files

File: README.md [hunk 1]
```diff
@@ -42,18 +42,13 @@ broken/working pages, or toggle Tracking Protection on or off:
 3. `npm install`
 4. `npm run bundle`
 
-### With `web-ext`
+This add-on depends on [`web-ext`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext). Using `web-ext`, start a version of Firefox running the add-on like so:
```
Comment (pdehaan) [inline README.md]: Not sure if we can just remove the `/en-US/` from the URL and let Mozilla serve up a translated version, if available.

File: README.md [hunk 2]
```diff
@@ -42,18 +42,13 @@ broken/working pages, or toggle Tracking Protection on or off:
...
-2. `./node_modules/.bin/web-ext run --source-dir=src --firefox-binary {path to Firefox 49+ binary}`
+`./node_modules/.bin/web-ext run --source-dir=src --firefox-binary {path to Firefox 49+ binary}`
 
-### Without `web-ext`
-
-1. Go to `about:config` and set `xpinstall.signatures.required` to `false`
-2. Go to `about:debugging`
-3. Click "Load Temporary Add-on"
-4. Select this repo's `src/manifest.json` file
+Add-on development is better with [a particular  environment](https://developer.mozilla.org/en-US/Add-ons/Setting_up_extension_development_environment). For convenience, this repository includes a vanilla Firefox profile that has the [DevPrefs addon](https://addons.mozilla.org/en-US/firefox/addon/devprefs/) installed. If you wish to use this profile, try the following command:
```
Comment (pdehaan) [inline README.md]: Same as above, if we can possibly remove /en-US/ from these 2 URLs and let them redirect.

File: README.md [hunk 3]
```diff
@@ -42,18 +42,13 @@ broken/working pages, or toggle Tracking Protection on or off:
...
 
+``./node_modules/.bin/web-ext run --source-dir=src --firefox-binary {path to Firefox 49+ binary} --firefox-profile ./dev_profile`
```
Comment (pdehaan) [inline README.md]: Has an extra leading "`" at the beginning of the line.

**Comment by pdehaan** [comment]
@hoosteeno I've never used a dev_profile before. What does it add? 
Is anything in the 208 new files specific to blok/Tracking Protection, or is this all just boilerplate? Is there anything we'd need to be aware of or potentially change in the future?

**Comment by hoosteeno** [comment]
@pdehaan The Firefox profile in the prior commit was a vanilla profile with the DevPrefs addon installed, which sets some sane defaults for addon development (like enabling Browser Toolbox). After talking with @groovecoder more about it, I agreed to ditch the committed profile and simply document how to create your own. This PR now does that.

File: README.md [hunk 4]
```diff
@@ -42,18 +42,25 @@ broken/working pages, or toggle Tracking Protection on or off:
...
+
+1. Make a new profile by running `{path to Firefox binary} -no-remote -P {new_profile_name} &`, which launches the profile editor. Name it whatever you wish (e.g. 'blok_dev'); store it in the default directory or in a directory of your choosing. Exit the profile editor.
+
+2. Launch your new profile with the very same command, so you can install the DevPrefs add-on: `{path to Firefox binary} -no-remote -P {new_profile_name} &`. Once Firefox launches, go to Tools->Add-ons and search for "DevPrefs". Install it. Close Firefox.
+
+3. Now you have a new, vanilla Firefox profile with the DevPrefs add-on installed. You can use your new profile with the code in _this_ repository like so:
 
+``./node_modules/.bin/web-ext run --source-dir=src --firefox-binary {path to Firefox 49+ binary} --firefox-profile {new_profile_name}`
```
Comment (pdehaan) [inline README.md]: Extra leading backtick.

**Comment by hoosteeno** [comment]
Alright, that was a bunch of fixup at the last minute, but I'm done.

**Comment by groovecoder** [comment]
LGTM.

**Pull request merged**